x-pack/plugins/security_solution/public/management/pages/policy/view/policy_hooks.ts

```diff
@@ -101,3 +101,17 @@ export function usePolicyDetailsArtifactsNavigateCallback(listId: string) {
     [getPath, history]
   );
 }
+
+export const useIsPolicySettingsBarVisible = () => {
+  const policyTabs = [
+    '/settings',
+    '/trustedApps',
+    '/eventFilters',
+    '/hostIsolationExceptions',
+    '/blocklists',
+  ];
+  return (
+    window.location.pathname.includes('administration/policy') &&
+    policyTabs.includes(window.location.pathname)
```

Comment:
❓ Why are you checking for the `policyTabs`? I think only the `/settings` uses the EuiBotomBar component, right?